docs/eth/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What is ETH?
```

Comment:
`What is Ether (ETH)?`

There's significant search volume around "ether", so I believe we should include mentions of that.